test/support/test_case.rb

```diff
@@ -16,10 +16,14 @@ class TestCase < Test::Unit::TestCase
 
     def setup
       @temp_file = nil
+      @original_root = Dir.home("root")
+      dir = Dir.mktmpdir
+      Dir.chroot(dir)
```

Comment:
@ko1  I think `Dir.chroot` requires privileged permission to call and I'm not able to find a replacement for it. Do you know any alternatives?

Reply:
I see.
@nobu had advised that changing `ENV['HOME']` is easy way for this kind of hack.